UnitTests/GitCommands.Tests/Git/GitCommandsHelperTest.TestGetStatusChangedFilesFromString.ForScenario.status_ignored_files.approved.json

```diff
@@ -5,17 +5,18 @@
         "TreeGuid": null,
         "RenameCopyPercentage": null,
         "Staged": 2,
-        "IsTracked": false,
+        "IsTracked": true,
         "IsDeleted": false,
-        "IsChanged": false,
+        "IsChanged": true,
```

Comment:
Why have these changed for ignored files?

Reply:
Because I updated the test case inputs: Many were identical previously

Reply:
@RussKie I separated the test case changes to a separate commit. Can you please review?